src/app/rmm.ts

```diff
@@ -36,6 +37,8 @@ export class RMM {
     public alias: Alias;
     public me: Me;
     public runbox_domain: RunboxDomain;
+    public tinymce_plugin: TinyMCEPlugin;
+    public plugins: any = {};
```

Comment:
I don't think this needs to exist if we only have one plugin. https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it

Also, what does the editor have to do with the API? This is just mixing concerns and creating a God-object like Webmail.pm – I thought we wanted to move away from that, otherwise we could've just sticked to existing rmmapi.

Reply:
Its like why there are powerplugs availables in everyhome if some of them are empty and not sold ? The constructors should let the owners do the powerplug themselfs instead of foreseen they will need some eventually ?

Reply:
I don't think this analogy holds. Powerplugs are hard to add later. Code is not. This is like adding an elevator to a single floor house because you may need more later.

Reply:
But more importantly, again: what does an editor do in an API object? Let's not create another Webmail.pm here.

Reply:
fixed